app/routers/match_router.py

```diff
@@ -0,0 +1,54 @@
+from fastapi import APIRouter
+
+from app.data import MongoDB
+from app.schema import MatchQuery, MatchUpdate
+
+Router = APIRouter(
+    tags=["Matching Operations"],
+)
+
+Router.db = MongoDB()
+
+
+@Router.post("/create/match")
+async def create_match(data: MatchUpdate):
+    """Insert a new mentor/mentee pairing into the collection
+    <pre><code>
+    @param data: JSON[MatchUpdate]
+    @return JSON[Boolean] - success or failure of update</pre></code>"""
+    data = data.dict()
+    return {"result": Router.db.upsert_to_set_array("Matches",
+                                                    {"mentor_id": data["mentor_id"]},
+                                                    {"mentee_ids": data["mentee_id"]})}
+
+
+@Router.post("/delete/match")
+async def delete_match(data: MatchUpdate):
+    """Remove a mentor/mentee pairing from the collection
+    <pre><code>
+    @param data: JSON[MatchUpdate]
+    @return JSON[Boolean] - success or failure of update</pre></code>"""
```

Comment:
small type here (i think it should say `@return JSON[Boolean] - success or failure of delete</pre></code>` Be careful of the copy/pasta bug ;)